### DISCUSSION TITLE
Why still beta?

### DISCUSSION THREAD

**Discussion by Anthchirp** [jab/bidict#208] [Q&A] [answered]
The package seems to have been around for a while, has everything what you'd expect from a stable package, the repository has nearly 1,000 stars, and there is even an enterprise support offer -- all that makes me wonder why you still class it as Beta?
https://github.com/jab/bidict/blob/fac42f4ce3cdceec16538992219494d9f97a5b9e/setup.py#L66

Do you expect the interface to undergo any future changes?
Or to pose the question in another way: what is missing for a 1.0 release? 🙂

**Comment by jab** [comment, marked as answer]
Better late than never:

For years I've been trying to recruit an additional maintainer, and even just a Python collections expert to do a one-time "1.0-readiness assessment" code + API review (even offering to pay for it out of the funds that TideLift has provided), but I haven't been able to find anyone interested yet.

For now, bidict is still almost entirely the result of the nights-and-weekends, "passion project" efforts of a single person (me). I'll keep trying to get the missing support and expert help that has been blocking a 1.0 release, and hopefully will finally succeed one of these days.

In the meantime, as a solo maintainer, it's been really helpful to be able to take advantage of the "v0" status to make e.g. [simplifications and performance improvements](https://bidict.readthedocs.io/en/v0.23.0/changelog.html) that are technically breaking, but that likely affect very few (if any) users. The other major changes I might like to try before going 1.0 are compiling a native speedup extension module, if it doesn't end up adding too much complexity, instability, or maintenance overhead.

Thanks for your interest, and if you or anyone you know would be able to help with any of the above, please let me know!

P.S. [ZeroVer](https://hynek.me/articles/semver-will-not-save-you/#:~:text=zerover) resonates really strongly here.
